contracts/ConnextModule.sol

```diff
@@ -63,17 +63,30 @@ contract ConnextModule is Module {
         emit ModuleSetUp(owner(), avatar, target, originAddress, origin, executor);
     }
 
-    modifier onlyExecutor(bytes memory _message) {
+    modifier onlyExecutor(address _originSender, uint32 _origin) {
```

Comment:
this will be coming directly from the `Connext` contract now instead of the `Executor`, we can update the modifier name to reflect that.